lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -467,12 +467,12 @@ my %stats = (
             my ($self, $sql) = @_;
 
             my $data = $sql->select_list_of_lists(<<~'SQL');
-                SELECT (cover_art_url ~ '^https?://.*.images-amazon.com')::int AS is_amazon, COUNT(*) FROM release_coverart
+                SELECT (cover_art_url ~ '^https?://.*.images-amazon.com')::int AS is_amazon, COUNT(*) FROM release_coverart_backup
```

Comment:
I don't check if the table exists here or in the report below, since these don't really need to run outside of production.